### PR TITLE
Feature/polish

### PR DESCRIPTION
**Pull request by ilonadynska**
## Description

Add polish localization to the project.

### CODE REVIEW

File: app/locales/pl.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    "push_at_risk_message": "Spotkałeś na swojej drodze osobę zakażoną COVID-19",
+    "push_at_risk_title": "Możesz być w niebezpieczeństwie",
+    "see_exposure_history": "Zobacz historię narażenia",
+    "settings_title": "Tablica",
+    "share_location_data": "Udostępnij dane o lokalizacji",
+    "show_overlap": "Kliknij aby wyświetlić zestaw danych publicznych",
+    "team": "Zespół",
+    "team_para": "Nasz zespół składa się z konsorcjum epidemiologów, inżynierów, naukowców, cyfrowych ewangelistów prywatności, profesorów i badaczy z uznanych instytucji, w tym: MIT, Hardvard, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young oraz Link Ventures.",
+    "terms_of_use": "Warunki korzystania",
+    "terms_of_use_url": "https://docs.google.com/document/d/1mtdal_pywsKZVMXLHjjj5eKznipPLP8sM1HwFTIhjo0/edit#",
```
Comment (tstirrat) [inline app/locales/pl.json:96]: Remove this key

**Comment by alpita-masurkar** [comment]
5 missing keys... please compare with en.json
home_setting_off_header,
home_setting_off_subtext,
terms_of_use_url
logging_active
logging_inactive

**Comment by ilonadynska** [comment]
@alpita-masurkar @tstirrat what to do with `terms_of_use_url`? remove key as requested above, so I think english will be used as fallback or need to add to polish but keeping the same value?

**Comment by alpita-masurkar** [comment]
> @alpita-masurkar @tstirrat what to do with `terms_of_use_url`? remove key as requested above, so I think english will be used as fallback or need to add to polish but keeping the same value?

remove the key altogether like @tstirrat suggested.

**Comment by ilonadynska** [comment]
ok, so the last commit should fix the missing keys and I removed key/value for `terms_of_use_url`

**Pull request merged**